File: LLHFunctionInterface.java
import java.security.Timestamp;
import java.util.List;

public interface LLHFunctionInterface {
    List<Integer> apply(List<Integer> orgTour);
    int getCounter();
    void incrementCounter();
    long getTimestamp();
    void setTimestamp(long timestamp);
    double getScore();
    void setScore(double score);
    Integer getConsec_counts(LLHFunctionInterface function);
    void setConsec_counts(LLHFunctionInterface function, Integer count);
    void addConsec_counts(LLHFunctionInterface function);
    double getF1();
    void setF1(double f1);
    void addF2(LLHFunctionInterface function,double f2);
    double getF2(LLHFunctionInterface function);
} 

File: tse/LLHFunctionInterface.cs
using System;
using System.Collections.Generic;

public interface  ILLHFunctionInterface
{
    List<int> Apply(List<int> orgTour);
    int GetCounter();
    void IncrementCounter();
    long GetTimestamp();
    void SetTimestamp(long timestamp);
    double GetScore();
    void SetScore(double score);
    int GetConsecCounts(ILLHFunctionInterface function);
    void SetConsecCounts(ILLHFunctionInterface function, int count);
    void AddConsecCounts(ILLHFunctionInterface function);
    double GetF1();
    void SetF1(double f1);
    void AddF2(ILLHFunctionInterface function, double f2);
    double GetF2(ILLHFunctionInterface function);
}
